orchestrate/spark_example/model.py

```diff
@@ -0,0 +1,53 @@
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Originally from https://github.com/apache/spark/blob/master/examples/src/main/python/pi.py
+# Edited to show use with Sigopt Orchestrate
+
+from __future__ import print_function
+
+import orchestrate.io
+import sys
+from random import random
+from operator import add
+
+from pyspark.sql import SparkSession
+
+
+if __name__ == "__main__":
+    """
+        Usage: pi [partitions]
+    """
+    spark = SparkSession\
+        .builder\
+        .appName("PythonPi")\
+        .getOrCreate()
+
+    partitions = int(sys.argv[1]) if len(sys.argv) > 1 else 2
+    n = 100000 * partitions
+
+    def f(_):
+        x = orchestrate.io.assignment('rand_num', 0.4) * 2 - 1
```

Comment:
since `rand_num` is the same here it should follow that the default values are the same. If you'd like to use different values for x and y it might work better to have parameters `x` and `y` (might also confuse our customers to call values returned by our API "random")